examples/tests/NftLinker.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
```

Comment:
do we even need these tests? the example should cover the correctness of the contract

Reply:
removed the tests